### ISSUE TITLE
install w/ python3 `distribute` issue

### ISSUE THREAD

**Issue by jklymak** [nucleic/kiwi#37]
Hi, 

This library depends on `distribute`, which works fine for python 2.7, but throws errors for python 3.6:

``` 
Collecting distribute
  Using cached distribute-0.7.3.zip
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
....
   AttributeError: module 'importlib._bootstrap' has no attribute 'SourceFileLoader'
```

Quick internet searches just say "too bad, you are using the wrong version of python".  Does kiwi really need `distribute`?

**Comment by jklymak** [comment]
ping @jwiggins who worked on the python 3 support.

**Comment by jwiggins** [comment]
Git blame for the `distribute` requirement: https://github.com/nucleic/kiwi/commit/5bc16c2f9692cc9ac0be5e655a01b7647b409c0e

I'm not sure why @sccolbert added it then. It's possible that it's not needed at all now.

**Comment by MatthieuDartiailh** [comment]
Closed by #22

**Comment by jklymak** [comment]
Works great now.  Thanks a lot @jwiggins  and  @MatthieuDartiailh!
